01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,5 @@
+var main = function() {
+    $('.hidden').fadeIn(5000);
+    $( '.alias' ).focus().delay(5000); /*can't set the delay*/
```

Comment:
Usually we add comments on top of the line (in case is an inline comment) with // instead of a block comment.